frontends/web/src/routes/new-settings/components/about/app-version-setting.module.css

```diff
@@ -0,0 +1,16 @@
+.version {
+    color: var(--color-secondary);
+    font-size: 14px;
+    margin: 0;
+    overflow: hidden;
+    text-overflow: ellipsis;
+}
+
+.versionContainer {
+    display: flex;
+    align-items: center;
+}
+
+.versionContainer > p {
```

Comment:
nit: does it need to do direct child, or could it just be in styles.version ?

Reply:
yeah.. good catch:)